src/prefect/schedules/clocks.py

```diff
@@ -390,8 +390,10 @@ class RRuleClock(Clock):
     Clock that supports iCal style RRule (recurrence rules).
 
     See below links for helpful info:
-        - https://icalendar.org/iCalendar-RFC-5545/3-8-5-3-recurrence-rule.html
-        - https://dateutil.readthedocs.io/en/stable/rrule.html
+    
+    IETF iCalendar specification RFC5545: https://icalendar.org/iCalendar-RFC-5545/3-8-5-3-recurrence-rule.html
+    
+    dateutil rrule module, an implementation of the iCal specification recurrence rules: https://dateutil.readthedocs.io/en/stable/rrule.html
```

Comment:
actually it looks like this broke `black` somehow - you'll need to run `black` to resolve

Reply:
Fixed.